pkg/util/shortid_generator.go

```diff
@@ -25,7 +26,17 @@ func IsShortUIDTooLong(uid string) bool {
 	return len(uid) > 40
 }
 
-// GenerateShortUID generates a short unique identifier.
+// GenerateShortUID will generate a UUID that can also be a k8s name
+// it is gaurenteed to have a character as the first letter
+// This will return a valid k8s name
 func GenerateShortUID() string {
-	return shortid.MustGenerate()
+	uid, err := uuid.NewRandom()
+	if err != nil {
+		panic("invalid uuid")
+	}
+	uuid := uid.String()
+	if unicode.IsDigit(rune(uuid[0])) {
```

Comment:
do we actually need unicode here? it seems like we could just declare alphaRunes as a string and do something like this (without the ambiguity of what happens if the string is not a digit but also not an allowed first character for some reason):
```suggestion
	if !strings.Contains(alpharunes, string(uuid[0])) {
```